account-kit/react/src/tailwind/plugin.ts

```diff
@@ -118,6 +118,13 @@ export const withAccountKitUi = (
   config: TailwindConfig,
   themeOverride?: AccountKitThemeOverride
 ): TailwindConfig => ({
+  darkMode: [
+    "variant",
+    [
+      "@media (prefers-color-scheme: dark) { &:not(.light, .light *) }",
+      "&:is(.dark, .dark *)",
+    ],
+  ],
```

Comment:
this is above the `...config` so that the client apps can override it

Reply:
does this also mean that dark mode must be enabled using the `dark` class?

Reply:
nope, it handles both! so the OS preference or the class-based approach will work